### PR TITLE
Improve CZ translation

### LINKED ISSUE #3: Translate programmatically
translate programmatically mlforkids-api/public/languages/cs.json

### PR DESCRIPTION
**Pull request by Michal-Miky-Jankovsky**
naive translation via DeepL free account

!!! need to be checked before merge to main repo !!!

todo:
 - [x] fix nonsense like Trenovat VS Vlak (EN Train)
 - [x] should `&` be replaced by `... a ...` in CZ

### CODE REVIEW

**Comment by Michal-Miky-Jankovsky** [comment]
find someone to check each record

File: mlforkids-api/public/languages/cs.json [hunk 1]
```diff
@@ -2,2295 +2,2284 @@
...
     },
     "ABOUT": {
         "TITLE": "Strojové učení pro děti",
         "IN6SECS": "Za 6 sekund...",
         "IN20MINS": "Za 20 minut...",
         "WHAT": {
-            "TITLE": "Co?",
-            "BODY": "This free tool introduces machine learning by providing hands-on experiences for training machine learning systems and building things with them. <br/><br/>It provides an easy-to-use guided environment for training machine learning models to recognise text, numbers, images, or sounds. <br/><br/>This builds on existing efforts to introduce and teach coding to children, by adding these models to educational coding platforms <a href='https://scratch.mit.edu/about' target='_blank'>Scratch</a> and <a href='http://appinventor.mit.edu/' target='_blank'>App Inventor</a>, and helping children create projects and build games with the machine learning models they train. <br/><br/>The tool was first made available in 2017, and is now being <a href='https://wakelet.com/wake/4d3680cb-2a11-49ab-9180-fcfeced48393' target='_blank'>used by thousands of schools, code clubs, and families around the world</a>. <br/><br/>To learn more, look at:"
+            "TITLE": "Cože?",
```
Comment (michal-jankovsky-ipf) [inline mlforkids-api/public/languages/cs.json]: Co?

File: mlforkids-api/public/languages/cs.json [hunk 2]
```diff
@@ -2,2295 +2,2284 @@
...
                     }
                 }
             }
         },
         "TRAINING": {
-            "TITLE": "Info from training computer:",
-            "STARTED": "Model started training at:",
-            "STATUS": "Current model status:",
+            "TITLE": "Informace ze školicího počítače:",
+            "STARTED": "Model zahájil školení na:",
```
Comment (janlouda) [inline mlforkids-api/public/languages/cs.json]: Trénování modelu začalo v

Comment (Michal-Miky-Jankovsky) [reply mlforkids-api/public/languages/cs.json]: fixed

**Pull request merged**